hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakeRewardCalculatorImpl.java

```diff
@@ -55,7 +55,7 @@ public long computePendingReward(
         // At this point all the accounts that are eligible for computing rewards should have a
         // staked to a node
         final var nodeId = account.stakedNodeIdOrThrow();
-        final var stakingInfo = stakingInfoStore.get(nodeId);
+        final var stakingInfo = stakingInfoStore.getOriginalValue(nodeId);
```

Comment:
Why do we want the original value here?

Reply:
In mono services it uses stakingInfo from state, so used getOriginalValue()